README.md

```diff
@@ -54,7 +54,54 @@ Same thing about `message` object - it implements [Message](https://core.telegra
 
 ## Webhooks
 
-If you are going to use [webhooks](https://core.telegram.org/bots/api#setwebhook) instead of [long polling](https://core.telegram.org/bots/api#getupdates), you need to implement your own webhook callbacks server. Take a look at [this repo](https://github.com/solyaris/BOTServer) as an example.
+If you are going to use [webhooks](https://core.telegram.org/bots/api#setwebhook) (and you should!) instead of [long polling](https://core.telegram.org/bots/api#getupdates), you need to implement your own webhook callbacks server.
+
+A Sinatra quick start example:
+
+1. Rake task for setting the webhook url for your bot:
+
+```ruby
+task :telegramwebhook do
+    require 'telegram/bot'
+    HOOK_URL = "https://<your server url>/telegram"
+    bot = Telegram::Bot::Api.new(ENV['TELEGRAM_BOT_API_TOKEN'])
+    puts bot.set_webhook(url: HOOK_URL)
+end
+```
+
+2. Post route for receiving the JSON data from Telegram
+
+```ruby
+API = "https://api.telegram.org/file/bot"
+
+post '/telegram' do
+    request.body.rewind
+    data = JSON.parse(request.body.read)
+    @bot = Telegram::Bot::Api.new(ENV['TELEGRAM_BOT_API_TOKEN'])
```

Comment:
Is it necessary to use instance variables for `bot` and `message`?

Reply:
No, of course not. But on the other hand it won't hurt anyone... ;-)